src/studio-home/create-new-course/index.jsx

```diff
@@ -0,0 +1,195 @@
+import { useCreateNewCourse } from './hooks';
+import messages from './messages';
+
+const CreateNewCourse = ({ handleOnClickCancel, handleOnClickCreate }) => {
+  const {
+    intl,
+    errors,
+    values,
+    postErrors,
+    isFormFilled,
+    isFormInvalid,
+    organizations,
+    showErrorBanner,
+    dispatch,
+    handleBlur,
+    handleChange,
+    hasErrorField,
+    setFieldValue,
+  } = useCreateNewCourse();
+  const savingStatus = useSelector(getSavingStatus);
+
+  const newCourseFields = [
+    {
+      label: intl.formatMessage(messages.courseDisplayNameLabel),
+      helpText: intl.formatMessage(messages.courseDisplayNameHelpText),
+      name: 'displayName',
+      value: values.displayName,
+      isDropdown: false,
+      placeholder: intl.formatMessage(messages.courseDisplayNamePlaceholder),
+    },
+    {
+      label: intl.formatMessage(messages.courseOrgLabel),
+      helpText: intl.formatMessage(messages.courseOrgHelpText, {
+        strong: (
+          <strong>
+            {intl.formatMessage(messages.courseOrgHelpTextStrong)}
+          </strong>
+        ),
+      }),
+      name: 'org',
+      value: values.org,
+      isDropdown: true,
+      options: organizations,
+      placeholder: intl.formatMessage(messages.courseOrgPlaceholder),
+    },
+    {
+      label: intl.formatMessage(messages.courseNumberLabel),
+      helpText: intl.formatMessage(messages.courseNumberHelpText, {
+        strong: (
+          <strong>
+            {intl.formatMessage(messages.courseNumberHelpTextStrong)}
+          </strong>
+        ),
+      }),
+      name: 'number',
+      value: values.number,
+      isDropdown: false,
+      placeholder: intl.formatMessage(messages.courseNumberPlaceholder),
+    },
+    {
+      label: intl.formatMessage(messages.courseRunLabel),
+      helpText: intl.formatMessage(messages.courseRunHelpText),
+      name: 'run',
+      value: values.run,
+      isDropdown: false,
+      placeholder: intl.formatMessage(messages.courseRunPlaceholder),
+    },
+  ];
+
+  const onClickSave = () => {
+    dispatch(updateNewCourseData(values));
+    handleOnClickCreate();
+  };
+
+  const onClickCancel = () => {
+    dispatch(updatePostErrors({}));
+    handleOnClickCancel();
+  };
+
+  const createButtonState = {
+    labels: {
+      default: intl.formatMessage(messages.createButton),
+      pending: intl.formatMessage(messages.creatingButton),
+    },
+    disabledStates: [STATEFUL_BUTTON_STATES.pending],
+  };
+
+  return (
+    <div className="new-course-section mb-4.5">
+      <TransitionReplace>
+        {showErrorBanner ? (
+          <AlertMessage
+            variant="danger"
+            icon={InfoIcon}
+            title={postErrors.errMsg}
+            aria-hidden="true"
+            aria-labelledby={intl.formatMessage(messages.alertErrorExistsAriaLabelledBy)}
+            aria-describedby={intl.formatMessage(messages.alertErrorExistsAriaDescribedBy)}
+          />
+        ) : null}
```

Comment:
What advantage does using a ternary expression give us in this case?

Reply:
When we use && we didn't have special effect from transition